bindata/v3.11.0/openshift-controller-manager/unidling-controller-clusterrole.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: system:openshift:openshift-controller-manager:unidling-controller
```

Comment:
@sttts @deads2k @soltysh 

To be honest I don't know which team _should_ own this.